renovate.json

```diff
@@ -21,8 +21,9 @@
   ],
   "rangeStrategy": "bump",
   "lockFileMaintenance": {
-    "enabled": false
+    "enabled": true,
+    "automerge": true
   },
-  "reviewers": ["Siegrift"],
+  "reviewers": ["Siegrift", "aquarat", "bdrhn9"],
```

Comment:
Maybe just @bdrhn9 as a reviewer? Btw. updating this warrants a change in technical documentation.

Reply:
Let's wait for Bedirhan to confirm he's OK with it.

Reply:
I'm okay with it.